rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -539,6 +539,52 @@ void setLockWhitelistDisableBlockDelay() throws BlockStoreException, IOException
         }
     }
 
+    @Nested
+    @Tag("LockingCap")
+    class LockingCapTest {
+
+        private LockingCapSupport lockingCapSupport;
+        private BridgeSupport bridgeSupport;
+        LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+
+        @BeforeEach
+        void setUp() {
+            lockingCapSupport = mock(LockingCapSupportImpl.class);
+            bridgeSupport = bridgeSupportBuilder
+                .withLockingCapSupport(lockingCapSupport)
+                .build();
+        }
+
+        @Test
+        void getLockingCap() {
```

Comment:
should these test names be more descriptive?

Reply:
I agree with @apancorb. It does not hurt to use a name like you did in the other tests. For instance: `getLockingCap_whenPreviousLockingCapExist_shouldReturnExisitngLockingCap`

Reply:
We can speak about that because this is a kind of standard with the tests related to refactors in `BridgeSupportTest`. You can take a look at previous refactor methods as I pointed it out previously.